src/main/java/io/swagger/inflector/models/ApiError.java

```diff
@@ -16,10 +16,20 @@
 
 package io.swagger.inflector.models;
 
+import javax.ws.rs.core.Response;
```

Comment:
can we remove this import, if it's just used to get the Response.Status.INTERNAL_SERVER_ERROR code and just use 500 as an integer?